.github/workflows/aselo_beta.yml

```diff
@@ -64,14 +64,49 @@ jobs:
         with:
           ssm_parameter: "PROD_TWILIO_AS_OPERATING_INFO_KEY"
           env_variable_name: "OPERATING_INFO_KEY"
-      - name: Set serverless base URL
+      - name: Set Twilio Flex Proxy service ID
         uses: "marvinpinto/action-inject-ssm-secrets@latest"
         with:
-          ssm_parameter: "ASELO_BETA_SERVERLESS_BASE_URL"
-          env_variable_name: "SERVERLESS_BASE_URL"          
-      # Call main-action to compile, deploy and check availability
-      - name: Executing main-action
-        uses: ./.github/actions/main-action
+          ssm_parameter: "PROD_TWILIO_AS_FLEX_PROXY_SERVICE_SID"
```

Comment:
Note that Aselo Beta is actually using `Production` environment. I think we can leave it as it is for now but it will be nice to change the variables on Parameter Store at some point.

Reply:
Yes, it would be good to change that